modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/java/com/liferay/frontend/data/set/views/web/internal/portlet/FDSViewsPortlet.java

```diff
@@ -802,20 +886,33 @@ private synchronized void _generate(
 			long companyId, Locale locale, long userId)
 		throws Exception {
 
-		ObjectDefinition fdsEntryObjectDefinition =
+		ObjectDefinition fdsViewObjectDefinition =
 			_objectDefinitionLocalService.fetchObjectDefinition(
-				companyId, "FDSEntry");
+				companyId, "FDSView");
 
-		if (fdsEntryObjectDefinition != null) {
+		if (fdsViewObjectDefinition != null) {
 			return;
 		}
 
-		fdsEntryObjectDefinition = _createFDSEntryObjectDefinition(
-			locale, userId);
+		if (FeatureFlagManagerUtil.isEnabled("LPD-15729")) {
```

Comment:
We're hiding new persistence behind FF. Removing this FF will break everything. 

Important note though, **this PR will break all tests locally**. Locally, everybody will need to add this FF. CI should be fine, as we're adding new FF to the tests `env`.

Reply:
we can live with it :)